tgbot.py

```diff
@@ -7,9 +7,32 @@
 import requests
 import sqlite3
 import sql
+import configparser
+
+def CheckConfig():
+        '''
+        ChekConfig() funtion return the token from config.ini file
+        it will return False if token is empty or config.ini file structure changed
+        '''
+        config = configparser.ConfigParser()
+        config.read('config.ini')
+        TOKEN=""
+        try:
+                token=config['telegram']['token']
+                
+                if token=="" or token=="TOKEN_KEY_WITHOUT_QUOTES":
+                        print("Token Should not be Empty please add it in config.ini file")
+                        return False
+                elif '"' in token or "'" in token:
+                        print("Do not add Quotes to token use direct value\ntoken=XXXX:XXXXXX")
+                        return False
+                        
+                else:
+                        return token
+        except:
+                print('''Do not change Syntax of config.ini file It should be Like\n[telegram]\ntoken="YOUR TOKEN"''')
```

Comment:
change it to: `print('Do not change Syntax of config.ini file!')`

Reply:
@nileshpatra Completed